src/web/tokenserver.rs

```diff
@@ -67,13 +68,22 @@ pub struct Claims {
     pub sub: String,
     pub iat: i64,
     pub exp: i64,
+    #[serde(rename = "fxa-generation")]
+    pub generation: i64,
 }
 
 pub fn get(
+    req: HttpRequest,
     state: Data<ServerState>,
     auth: BearerAuth,
 ) -> impl Future<Output = Result<HttpResponse, BlockingError<ApiError>>> {
-    dbg!(&state.tokenserver_database_url);
+    let x_key_id = req
+        .headers()
+        .get("x-keyid")
```

Comment:
I'm wondering if we should add some error handling if `x-keyid` isn't included as a header or has an invalid value, kind of like how the python works below?